crds/configmanagement_v1_configmanagement.yaml

```diff
@@ -159,15 +160,19 @@ spec:
                     pattern: ^(ssh|cookiefile|gcenode|gcpserviceaccount|token|none)$
                     type: string
                   syncBranch:
-                    description: 'SyncBranch is the branch to sync from.  Default:
-                      "master".'
+                    description: SyncBranch is the git branch to sync from. When `spec.enableMultiRepo`
+                      is true, `syncBranch` defaults to using the git server's default
+                      branch, unless `syncRev` is set, then `syncRev` takes precedence
+                      over `syncBranch`. It is recommended to use the `syncRev` field.
+                      When `spec.enableMultiRepo` is false, `syncBranch` defaults
```

Comment:
This field cannot be set unless `spec.enableLegacyField` is enabled

Reply:
NVM, I just checked the code, `spec.enableLegacyField`  requires setting `spec.enableMultiRepo` to true.